### ISSUE TITLE
Hybrid configurations in Nrao.query_region

### ISSUE THREAD

**Issue by rgalvanmadrid** [astropy/astroquery#1282]
Hi. 

Adding a  hybrid VLA configuration in Nrao.query_region, such as 'BnA' doesn't work.  Below my attempt to reporting a traceback: 

```
In [31]: run traceback_error_1.py
Traceback (most recent call last):
  File "/home/roberto/ALMA_IMF/freefree/traceback_error_1.py", line 12, in <module>
    telescope_config=['A','AB','BnA'])
  File "/home/roberto/anaconda3/lib/python3.6/site-packages/astroquery/utils/class_or_instance.py", line 25, in f
    return self.fn(obj, *args, **kwds)
  File "/home/roberto/anaconda3/lib/python3.6/site-packages/astroquery/utils/process_asyncs.py", line 26, in newmethod
    response = getattr(self, async_method_name)(*args, **kwargs)
  File "/home/roberto/anaconda3/lib/python3.6/site-packages/astroquery/nrao/core.py", line 400, in query_region_async
    cache=cache)
  File "/home/roberto/anaconda3/lib/python3.6/site-packages/astroquery/nrao/core.py", line 342, in query_async
    request_payload = self._args_to_payload(**kwargs)
  File "/home/roberto/anaconda3/lib/python3.6/site-packages/astroquery/nrao/core.py", line 46, in wrapper
    .format(Nrao.telescope_config))
ValueError: 'telescope_config' must be one of ['ALL', 'A', 'AB', 'BnA', 'B', 'BC', 'CnB', 'C', 'CD', 'DnC', 'D', 'DA']
```

**Comment by bsipocz** [comment]
Hi @rgalvanmadrid!
Could you also copy paste the content of `traceback_error_1.py`?

**Comment by rgalvanmadrid** [comment]
import traceback

from astroquery.nrao import Nrao
import astropy.units as u
import astropy.coordinates as coord
from astropy.table import Table
import numpy as np

try:
	C_result_table = Nrao.query_region(coord.SkyCoord('19h23m44.1800s','+14d30m29.500s', \
	frame='icrs'), radius=3.75*u.arcmin, telescope='jansky_vla', obs_band='C', \
	telescope_config=['A','AB','BnA'])
except Exception as ex:
	print(''.join(traceback.format_exception(etype=type(ex), value=ex, tb=ex.__traceback__)))

**Comment by bsipocz** [comment]
This has been fixed by #1283
